common/datavalues/src/arrays/ops/vec_hash.rs

```diff
@@ -81,6 +81,7 @@ impl VecHash for DFStringArray {
     fn vec_hash(&self, hasher: DFHasher) -> Result<DFUInt64Array> {
         Ok(self.apply_cast_numeric(|v| {
             let mut h = hasher.clone_initial();
+            // Hash::hash_slice(v,&mut h);
```

Comment:
No used any more?

Reply:
datavalues will be removed, so I just add a comment that the previous usage is wrong.